internal/report/report.go

```diff
@@ -413,6 +414,9 @@ func PrintAssembly(w io.Writer, rpt *Report, obj plugin.ObjTool, maxFuncs int) e
 		}
 	}
 
+	// demangle symbols by default for disassembly report
+	options := []demangle.Option{demangle.NoParams, demangle.NoTemplateParams}
```

Comment:
I would've preferred to have a way to control this, (eg with symbolize=demangle=xxx). Sometimes having the mangled name is useful